src/main/java/it/pagopa/pm/gateway/controller/PostePayPaymentTransactionsController.java

```diff
@@ -125,6 +122,7 @@ public ResponseEntity<ACKMessage> updatePostePayTransaction(@RequestBody AuthMes
             requestEntity.setIsProcessed(true);
             requestEntity.setAuthorizationOutcome(isAuthOutcomeOk);
             requestEntity.setAuthorizationCode(authCode);
+            requestEntity.setStatus(isAuthOutcomeOk?PaymentRequestStatusEnum.AUTHORIZED.name():PaymentRequestStatusEnum.DENIED.name());
```

Comment:
@andrearovere why are you storing the enum name in the database?
Is this a requirement? If not, I think it is maybe better if you store the Long value

P.S. remember to reformat your code (ctrl + alt + l) to make it more readable

Reply:
Yes it's a requirement discussed with Mario and Simone